bin/db-helper

```diff
@@ -99,7 +110,7 @@ function create_clean_file() {
     local tables=`cat $ddl_file | grep -i "create table" | awk ' { print $3 } '`
 
     local tmp="/tmp/clean-$DATABASE.$$"
-    echo "/*! use $DATABASE; */" >> $tmp
+    echo "/*! USE $DATABASE; */" >> $tmp
```

Comment:
That syntax does not seem to work for me (when using mysql on macos). I get the following error:
`ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 1`

However if i just specify;
`use $database` without the comment that works. I am unclear if this was added for postgreSQL support?